src/main/java/name/valery1707/kaitai/KaitaiMojo.java

```diff
@@ -128,7 +123,15 @@
 	/**
 	 * Executes the plugin, to read the given source and behavioural properties and generate POJOs.
 	 */
-	public void execute() throws MojoExecutionException, MojoFailureException {
+	public void execute() throws MojoExecutionException {
+		try {
+			executeInt();
+		} catch (KaitaiException e) {
+			throw new MojoExecutionException(e.getMessage(), e.getCause());
```

Comment:
\[Checkstyle\] WARNING: 'catch' child has incorrect indentation level 24, expected level should be 6\.